build-system/pr-check/dist-bundle-size.js

```diff
@@ -28,25 +28,31 @@ const {
   processAndUploadDistOutput,
   startTimer,
   stopTimer,
+  timedExecWithError,
   timedExecOrDie: timedExecOrDieBase,
   uploadDistOutput,
   verifyBranchCreationPoint,
 } = require('./utils');
 const {determineBuildTargets} = require('./build-targets');
 const {isTravisPullRequestBuild} = require('../travis');
 const {runYarnChecks} = require('./yarn-checks');
+const {signalDistUpload} = require('../tasks/pr-deploy-bot-utils');
 
 const FILENAME = 'dist-bundle-size.js';
 const FILELOGPREFIX = colors.bold(colors.yellow(`${FILENAME}:`));
 const timedExecOrDie = (cmd, unusedFileName) =>
   timedExecOrDieBase(cmd, FILENAME);
 
+function stopJob(startTime) {
+  stopTimer(FILENAME, FILENAME, startTime);
+  process.exitCode = 1;
+  return;
+}
```

Comment:
Nice! As a follow up, maybe move this function to `utils.js` and replace [all these instances](https://github.com/ampproject/amphtml/search?utf8=%E2%9C%93&q=%22stopTimer(FILENAME%2C+FILENAME%2C+startTime)%22) too?